src/components/GenerativeDirectAnswer.tsx

```diff
@@ -134,7 +137,12 @@ function Answer(props: AnswerProps) {
   </>;
 }
 
-interface CitationsProps {
+/**
+ * Props for citations component.
+ *
+ * @public
+ */
+export interface CitationsProps {
   gdaResponse: GenerativeDirectAnswerResponse,
```

Comment:
add docs for each of the fields now that this interface is public

Reply:
Added. Not showing as "outdated" because I edited lines below it